File: C#Basic/HomeWorks/Homework-Console-Input-Output/QuadraticEquation.cs
using System;

class QuadraticEquation
{
    static void Main()
    {
        Console.Write("Insert a= ");
        double a = float.Parse(Console.ReadLine());
        while (a == 0)
        {
            Console.WriteLine("\"a\", can not be 0");
            Console.Write("Insert a= ");
            double.TryParse(Console.ReadLine(),out a);
        }
        Console.Write("Insert b= ");
        double b = double.Parse(Console.ReadLine());
        Console.Write("Insert c= ");
        double c = double.Parse(Console.ReadLine());
        double d = Math.Pow(b, 2) - (4 * a * c);
        if (d < 0)
        {
            Console.WriteLine("No real roots");
        }
        else if (d == 0)
        {
            double x = (-b / (2 * a));
            Console.WriteLine("X1 = X2 = " + x);
        }
        else
        {
            double x1 = (-b - Math.Sqrt(d)) / (2 * a);
            double x2 = (-b + Math.Sqrt(d)) / (2 * a);
            Console.WriteLine("X1 = " + x1);
            Console.WriteLine("X2 = " + x2);
        }
    }
}

File: Advanced Level/JavaScriptBasics/HomeWorks/HW2/Problem1/quadratic equation.js
function quadratic_equation(a, b, c) {
    var a, b, c;

    var d = Math.pow(b, 2)-(4 * a * c);
    var x1 = (-b + Math.sqrt(d))/(2*a);
    var x2 = (-b - Math.sqrt(d))/(2*a);

    if (d < 0) {
        console.log("No real roots");
    } else if (d == 0) {
        console.log("X = "  + x1);
    } else {
        console.log("X1 = " + x1 + "\nX2 = " + x2);
    }
}
quadratic_equation(3,4,5);
quadratic_equation(2,5,-3);
quadratic_equation(2,-4,2);
quadratic_equation(4,2,1);